### ISSUE TITLE
Version 0.5.0 is broken

### ISSUE THREAD

**Issue by FM-96** [tom-s/clap-detector#9]
Attempting to install the newest version and run the example given in the readme leads to this error:

```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'lodash/uniqueid'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/[redacted]/node_modules/clap-detector/lib/index.js:26:40)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

The reason for the error is that [on line 36 of index.js](https://github.com/tom-s/clap-detector/blob/bd8d242f83368a1b1f31774778b8cdcca2a4222c/src/index.js#L36) the module is imported from `'lodash/uniqueid'` instead of `'lodash/uniqueId'` (note the capitalization).

**Comment by tom-s** [comment]
Yes sorry I did it on a Mac which doesn't care about the case. I'll fix it and let you know

**Comment by tom-s** [comment]
I've published version 5.0.1 which should solve the issue. Thank you for reporting it !
